lib/utils.ts

```
@@ -0,0 +1,311 @@
+    return "Today";
+  } else if (diffDays === 1) {
+    return "1 day ago";
+  } else {
+    return `${diffDays} days ago`;
+  }
+}
+
+export const createIframeLink = (videoId: string) =>
+  `https://iframe.mediadelivery.net/embed/421422/${videoId}?autoplay=true&preload=true`;
+
+export const doesTitleMatch = (videos: any, searchQuery: string) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)